libdnf.spec

```diff
@@ -23,7 +23,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.3
+Version:        0.8.0
```

Comment:
No, it's an API change so according to our previous discussions we have to bump minor not patch level.